src/main/java/oss/fosslight/api/controller/v1/ApiProjectController.java

```diff
@@ -487,7 +489,17 @@ public CommonResult ossReportSrc(
 						}
 						
 						UploadFile bean = apiFileService.uploadFile(ossReport); // file 등록 처리 이후 upload된 file정보를 return함.
-						Map<String, Object> result = apiProjectService.getSheetData(bean, prjId, "SRC");
+
+						// get Excel Sheet name starts with SRC
+						List<String> sheet = null;
+						try {
+							sheet = ExcelUtil.getSheetNameStartsWith("SRC", Arrays.asList(bean),
+									CommonFunction.emptyCheckProperty("upload.path", "/upload"));
+						}  catch (Exception e) {
+							e.printStackTrace();
```

Comment:
Let's change it to print by using log.
`log.error(e.getMessage(), e);`

Reply:
@soimkim 
It will be reflected.